EIPS/eip-4841.md

```diff
@@ -0,0 +1,420 @@
+---
+eip: 4841
+title: Expandable Onchain SVG Images Storage
+description: Expandable SVG images storage model for storing larger-capacity images on Ethereum.
+author: KyungEun Kim <kkekke815@gmail.com>, Soohan Park <725psh@gmail.com>, Soohan Park (@Soohan-Park)
+discussions-to: https://ethereum-magicians.org/t/eip-4841-expandable-onchain-svg-images-storage-structure/8410
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-02-23
+---
+
+## Abstract
+
+This standard proposal is an Expandable Onchain SVG Images Storage on Ethereum that permanently preserves images and prevents tampering, and can store larger-capacity images furthermore.
+
+It is a structure designed to store SVG images with a larger capacity by distributing SVG images in units of tags on Ethereum.
+
+The structure presented by this EIP consists of a total of three layers as shown below.
+
+![StructureDiagram.jpg](../assets/eip-4841/StructureDiagram.jpg)
+
+> **Storage Layer ─** A contract layer that stores distributed SVG images by tags.  
```

Comment:
It might be helpful to define "tag". Do you mean XML tag, or some sort of category tag?

Reply:
Actually, it means "XML tag" :)
I recognized the sentence could be confused, so I will refine it.
Thanks a lot!